src/utils/getValitadionSchemaLoginForm.js

```diff
@@ -0,0 +1,22 @@
+import * as yup from "yup";
+
+export const getValitadionSchemaLoginForm = () => {
+  const emailRegExp =
```

Comment:
I hope these regexps are the same as backend has

Reply:
yes